tool/tsh/mfa.go

```diff
@@ -378,6 +391,22 @@ func promptU2FRegisterChallenge(ctx context.Context, proxyAddr string, c *proto.
 	}}}, nil
 }
 
+func promptWebauthnRegisterChallenge(ctx context.Context, proxyAddr string, cc *wantypes.CredentialCreation) (*proto.MFARegisterResponse, error) {
+	// TODO(codingllama): Make sure users get a reasonable error back when the
+	//  origin doesn't match the RPID. Webauthn isn't as flexible as U2F in this
+	//  regard.
+	origin := proxyAddr
+	if !strings.HasPrefix(proxyAddr, "https://") {
+		origin = "https://" + origin
+	}
+	log.Debugf("WebAuthn: prompting U2F devices with origin %q", origin)
+
+	fmt.Println("Tap your *new* security key")
+
+	resp, err := wancli.Register(ctx, origin, wanlib.CredentialCreationFromProto(cc))
```

Comment:
```suggestion
	resp, err := wancli.Register(ctx, origin, wanlib.CredentialCreationFromProto(cc))
	if err != nil {
	    return nil, trace.Wrap(err)
	}
	return resp, nil
```

Reply:
I prefer the shorter form, which I've been pushing people to do as well, so I'll keep it.